mev_inspect/classifiers/specs/opensea.py

```diff
@@ -0,0 +1,44 @@
+from typing import List, Optional
+
+from mev_inspect.classifiers.helpers import create_nft_trade_from_transfers
+from mev_inspect.schemas.classifiers import ClassifierSpec, NftTradeClassifier
+from mev_inspect.schemas.nft_trades import NftTrade
+from mev_inspect.schemas.traces import DecodedCallTrace, Protocol
+from mev_inspect.schemas.transfers import Transfer
+
+OPENSEA_WALLET_ADDRESS = "0x5b3256965e7c3cf26e11fcaf296dfc8807c01073"
+
+
+class OpenseaClassifier(NftTradeClassifier):
+    @staticmethod
+    def parse_trade(
+        trace: DecodedCallTrace,
+        child_transfers: List[Transfer],
+    ) -> Optional[NftTrade]:
+        addresses = trace.inputs.get("addrs")
+        if addresses is None:
```

Comment:
When is this missing?

Reply:
Theoretically never. But the linter throws an error here if you don't check the None case - let me know if there's a better way around this

Reply:
Theoretically never. But the linter throws an error if you don't check the None case - is there a better way to handle?

Reply:
If you get the key explicitly it'll avoid the none type and just throw a KeyError if it's missing
```
addresses = trace.inputs["addrs"]
```